pkg/controller/nodedaemon/reconcile.go

```diff
@@ -57,22 +58,21 @@ func (r *DaemonReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
-	lvSets, tolerations, ownerRefs, nodeSelector, err := r.aggregateDeamonInfo(request)
```

Comment:
Are we deleting provisioner daemons that might have been running from before?

Reply:
added a commit for that